include/envoy/stats/stats.h

```diff
@@ -408,25 +408,33 @@ typedef std::unique_ptr<StoreRoot> StoreRootPtr;
 struct RawStatData;
 
 /**
- * Abstract interface for allocating a RawStatData.
+ * Abstract interface for allocating statistics. Alternate
```

Comment:
nit: remove `Alternate` since the interface doesn't impose a default.

Reply:
done